src/fragments/lib/auth/js/react-native-social.mdx

```diff
@@ -83,23 +83,23 @@ For React Native applications, you need to define a custom URL scheme for your a
 
  <plist version="1.0">
 
-     <dict>
-     <!-- YOUR OTHER PLIST ENTRIES HERE -->
-
-     <!-- ADD AN ENTRY TO CFBundleURLTypes for Cognito Auth -->
-     <!-- IF YOU DO NOT HAVE CFBundleURLTypes, YOU CAN COPY THE WHOLE BLOCK BELOW -->
-     <key>CFBundleURLTypes</key>
-     <array>
-         <dict>
-             <key>CFBundleURLSchemes</key>
-             <array>
-                 <string>myapp</string>
-             </array>
-         </dict>
-     </array>
-
-     <!-- ... -->
-     </dict>
+  <dict>
```

Comment:
Just wanna double check if we have a separate standard for markup or if it falls under the same one

Reply:
another good question for @rachelnabors